test/integration/short/metadata-tests.js

```diff
@@ -9,1090 +9,1132 @@ var errors = require('../../../lib/errors');
 var vit = helper.vit;
 var vdescribe = helper.vdescribe;
 
-describe('Metadata', function () {
+describe('metadata', function () {
```

Comment:
Are the changes in these tests mostly just for formatting?  Couldn't detect anything otherwise but thought i'd ask.

Reply:
Its lowercased, now the structure is the following:
- metadata
-- Metadata
-- Client#getState()